### PR TITLE
Fix #185

### LINKED ISSUE #185: Missing r-Value in /germany Response
Today I got an error, because the "value" fiels in the https://api.corona-zahlen.org/germany Response was missing. It just shows a date field. Could you please fix it?

### PR DESCRIPTION
**Pull request by PatrickHaussmann**
RKI renamed the column in the Excel file

### CODE REVIEW

**Review by Rubber1Duck — CHANGES_REQUESTED** [review]
RKI changed this very often! in the past they switched between 
"Punktschätzer des 4-Tage R-Wertes" and "Punktschätzer der 4-Tage R-Wert".
and i'm afraid they change back tomorrow and remove the "-".
So the better way should be

`let rValue = latestEntry["Punktschätzer des 4-Tage R-Wertes"] || latestEntry["Punktschätzer der 4-Tage R-Wert" || latestEntry["Punktschätzer des 4-Tage-R-Wertes"];`

**Review by Rubber1Duck — APPROVED** [review]

File: src/data-requests/r-value.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ function parseRValue(data: ArrayBuffer): { r: number, date: Date } | null {
 
     const latestEntry = json[json.length - 1];
     const dateString = latestEntry["Datum des Erkrankungsbeginns"];
-    let rValue = latestEntry["Punktschätzer des 4-Tage-R-Wertes"] || latestEntry["Punktschätzer der 4-Tage R-Wert"];
+    let rValue = latestEntry["Punktschätzer des 4-Tage R-Wertes"] || latestEntry["Punktschätzer der 4-Tage R-Wert" || latestEntry["Punktschätzer des 4-Tage-R-Wertes"];
```
Comment (Rubber1Duck) [inline src/data-requests/r-value.ts:13]: ups, there is a ] missing!
let rValue = latestEntry["Punktschätzer des 4-Tage R-Wertes"] || latestEntry["Punktschätzer der 4-Tage R-Wert"] || latestEntry["Punktschätzer des 4-Tage-R-Wertes"];

**Review by Rubber1Duck — CHANGES_REQUESTED** [review]

File: src/data-requests/r-value.ts (hunk 1 above)
Comment (PatrickHaussmann) [reply src/data-requests/r-value.ts:13]: sorry, I overlooked that.

Comment (Rubber1Duck) [reply src/data-requests/r-value.ts:13]: I suggested that also wrong! so thats my misstake!

**Review by Rubber1Duck — APPROVED** [review]
Now its working well!

**Review by marlon360 — APPROVED** [review]
Thanks @PatrickHaussmann

**Pull request merged**